src/ripple/basics/impl/PerfLogImp.cpp

```diff
@@ -80,26 +80,26 @@ PerfLogImp::Counters::countersJson() const
     Rpc totalRpc;
     for (auto const& proc : rpc_)
     {
-        Rpc const* value;
+        Rpc value;
         {
             std::lock_guard lock(proc.second.mutex);
             if (!proc.second.value.started && !proc.second.value.finished &&
                 !proc.second.value.errored)
             {
                 continue;
             }
-            value = &proc.second.value;
+            value = proc.second.value;
         }
 
         Json::Value p(Json::objectValue);
-        p[jss::started] = std::to_string(value->started);
-        totalRpc.started += value->started;
-        p[jss::finished] = std::to_string(value->finished);
-        totalRpc.finished += value->finished;
-        p[jss::errored] = std::to_string(value->errored);
-        totalRpc.errored += value->errored;
-        p[jss::duration_us] = std::to_string(value->duration.count());
-        totalRpc.duration += value->duration;
+        p[jss::started] = std::to_string(value.started);
+        totalRpc.started += value.started;
+        p[jss::finished] = std::to_string(value.finished);
+        totalRpc.finished += value.finished;
+        p[jss::errored] = std::to_string(value.errored);
+        totalRpc.errored += value.errored;
+        p[jss::duration_us] = std::to_string(value.duration.count());
+        totalRpc.duration += value.duration;
```

Comment:
This looks good--I made my previous comments while you were working on this fix.